pkg/console/operator/status.go

```diff
@@ -200,6 +200,14 @@ func (c *consoleOperator) ConditionDegraded(operatorConfig *operatorsv1.Console,
 	return operatorConfig
 }
 
+func (c *consoleOperator) IsConditionDegraded(operatorConfig *operatorsv1.Console, reason string) bool {
+	degradedCondition := v1helpers.FindOperatorCondition(operatorConfig.Status.Conditions, operatorsv1.OperatorStatusTypeDegraded)
+	if degradedCondition.Status == operatorsv1.ConditionTrue && degradedCondition.Reason == reason {
```

Comment:
I think there can be a null pointer error here if `FindOperatorCondition` cannot find the condition.  The function returns `nil` and then you check on `degradedCondition.Status` which does not exist.
The logs show:
```
Observed a panic: "invalid memory address or nil pointer dereference"
...
/go/src/github.com/openshift/console-operator/_output/local/go/src/github.com/openshift/console-operator/pkg/console/operator/status.go:204    <-------------
/go/src/github.com/openshift/console-operator/_output/local/go/src/github.com/openshift/console-operator/pkg/console/operator/sync_v400.go:92
```